app/models/ems_event.rb

```diff
@@ -178,6 +178,8 @@ def parse_event_metadata
       event_type == "datawarehouse_alert" ? message : nil,
       full_data.try(:[], :severity),
       full_data.try(:[], :url),
+      full_data.try(:[], :ems_ref),
+      full_data.try(:[], :resolved),
```

Comment:
The `try` logic is getting ugly.  It makes more sense to initialize `full_data` at the top of the method if it is `nil` then just reference the hash normally.

```ruby
def parse_event_metadata
  full_data ||= {}
  [
    event_type == "datawarehouse_alert" ? message : nil,
    full_data.[:severity],
    full_data[:url],
    full_data[:ems_ref],
    full_data[:resolved]
  ]
end
```